### PR TITLE
Include prisma_api_url and keys

### PR DESCRIPTION
**Pull request by Juliandreslopez**

### CODE REVIEW

File: action.yml [hunk 1]
```diff
@@ -155,3 +174,6 @@ runs:
     BITBUCKET_TOKEN: ${{ inputs.bitbucket_token }}
     BITBUCKET_USERNAME: ${{ inputs.bitbucket_username }}
     BITBUCKET_APP_PASSWORD: ${{ inputs.bitbucket_app_password }}
+    PRISMA_API_URL: ${{ inputs.PRISMA_API_URL }}
```
Comment (gruebel) [inline action.yml:177]:
```suggestion
    PRISMA_API_URL: ${{ inputs. prisma_api_url }}
```

**Review by gruebel — COMMENTED** [review]
hey, thanks for the contribution 💪 small problem, the Prisma credentials can't be set like that, they need to be passed to the `api-key` field.

**Comment by Saarett** [comment]
@Juliandreslopez Hey, are you still planning on fixing the issue?

File: action.yml [hunk 2]
```diff
@@ -155,3 +174,6 @@ runs:
...
+    PRISMA_ACCESS_KEY: ${{ secrets.PRISMA_CLOUD_ACCESS_KEY }}
```
Comment (billyjbryant) [inline action.yml:178]: I would refrain from hard-coding a requirement on `secrets` into the action and instead take required data as an input where a secret can be used if so desired. Additionally, the prisma api key and secret are ingestible already via the `api-key` input in the format of `<ACCESS_KEY>::<SECRET_KEY>`.

**Comment by Saarett** [comment]
Closing this PR due to inactivity. If you wish to contribute this feature, feel free to re-open or clone the PR.

**Pull request closed without merging**